www/_posts/2018-02-23-cordova-android-7.1.0.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Cordova Android 7.1.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `Cordova Android 7.1.0` has been released! This release fixes various bugs releated to the previous `cordova-android@7.0.0` release.
+
+To upgrade:
+
+    npm install -g cordova
+    cd my_project
+    cordova platform remove android
+    cordova platform add android@7.1.0
+
+To add it explicitly:
+
+    cordova platform add android@7.1.0
+
+<!--more-->
+## Curated Changelog
+* [CB-13879](https://issues.apache.org/jira/browse/CB-13879) updated gradle tools dependency to 3.0.1 for project template
+* [CB-13831](https://issues.apache.org/jira/browse/CB-13831) Update `android-versions` to 1.3.0 to support SDK 27.
```

Comment:
I know this is all auto-generated from commit messages, but for the sake of the blog post can we add what Android version SDK 27 corresponds to? I'm guessing that's 8.1?